src/app/game.ts

```diff
@@ -363,7 +363,14 @@ export class Game {
     }
   }
 
-  private stopGameLoop() {
+  unpauseGame() {
+    this.intervalId = window.setInterval(
+      () => this.update(),
+      GAME_LOOP_DELAY_IN_MILISECONDS
+    )
+  }
+
+  stopGameLoop() {
     if (this.intervalId !== undefined) {
       window.clearInterval(this.intervalId!)
```

Comment:
we dont need '!' at the end of intervalId. We already checked if its not udefined. we could actually get rid of if statement and then we can have this.intervalId! 
Im not even sure that we still need "!". it wont throw an error there if intervalId is undefined